ngraph/python/src/pyngraph/node.cpp

```diff
@@ -18,15 +18,27 @@
 #include "pyngraph/rt_map.hpp"
 #include "pyngraph/variant.hpp"
 
+class PyNode : public ngraph::Node {
+public:
+    std::shared_ptr<ngraph::Node> clone_with_new_inputs(const ngraph::OutputVector& inputs) const override {
+        PYBIND11_OVERRIDE_PURE(std::shared_ptr<ngraph::Node>, ngraph::Node, clone_with_new_inputs, inputs);
+    }
+
+    const type_info_t& get_type_info() const override {
+        PYBIND11_OVERRIDE_PURE(type_info_t&, ngraph::Node, get_type_info, );
```

Comment:
Not sure but is the last trailing comma needed here?

Reply:
It follows the convention from pybind's documentation, as in examples this is the way to inform that no arguments are used in underlying function. I can be kept in my opinion.